Bamboo/frontend/bamboo-frontend/app/routes/_setup+/welcome.tsx

```diff
@@ -30,6 +54,11 @@ export default function SetupWelcomePage(){
   const handleButtonClick = () => {
     if(selectState){
       console.log(`SUBMIT RESPONSE HERE!, ${selectState}`)
+      auth.initilaise.accountType(
+        BACKEND,
+        selectState,
+        jwt
```

Comment:
iirc http-only cookies will be sent to the backend if we set "auth:true" when making a request

Reply:
Set to be HTTP only "false" for debugging